scripts/fapi.py

```diff
@@ -0,0 +1,32 @@
+import uvicorn
```

Comment:
Need to add `uvicorn` to the requirements?

Reply:
It's not needed for startup, so as a development-only thing we'd probably move it in dev-requirements ?